packages/ckeditor5-alignment/src/utils.js

```diff
@@ -44,3 +50,60 @@ export function isDefault( alignment, locale ) {
 		return alignment === 'left';
 	}
 }
+
+/**
+ * Brings the configuration to the common form, an array of objects.
+ *
+ * @param {Array.<String|Object>} configuredOptions Alignment plugin configuration.
+ * @returns {Array.<Object>} Normalized object holding the configuration.
+ */
+export function normalizeAlignmentOptions( configuredOptions = [] ) {
+	const normalizedOptions = configuredOptions
+		.map( option => {
+			let optionObj;
+
+			if ( typeof option == 'string' ) {
+				optionObj = { name: option };
+			} else {
+				optionObj = { ...optionNameToOptionMap[ option ], ...option };
```

Comment:
This is confusing, what is it supposed to do?

Reply:
Default value from `optionNameToOptionMap[ option ]` gets overwritten by `option`.

Reply:
but `option` is not a string in this case so what it will take from the optionNameToOptionMap?

Reply:
Ah! Fixed!